api/src/main/java/jakarta/enterprise/inject/build/compatible/spi/BuildCompatibleExtension.java

```diff
@@ -22,13 +22,20 @@
  * on the particular processing phase and is documented in the corresponding extension annotation.
  * All the parameters will be provided by the container when the extension is invoked.
  * <p>
- * Extension methods can be assigned a priority using {@link ExtensionPriority @ExtensionPriority}.
- * Note that priority only affects order of extensions in a single phase.
+ * Extension methods can be assigned a priority using {@link jakarta.annotation.Priority @Priority}.
+ * Extension methods with smaller priority values are invoked first. Extension methods without specified priority
+ * have a default priority of {@link jakarta.interceptor.Interceptor.Priority#APPLICATION Priority.APPLICATION} + 500.
+ * If two extension methods have equal priority, the ordering among them is undefined. Note that priority
+ * only affects order of extensions in a single phase.
  * <p>
  * If the extension declares multiple methods, they are all invoked on the same instance of the class.
  * <p>
  * Extension classes can be annotated {@link SkipIfPortableExtensionPresent @SkipIfPortablExtensionPresent}
  * when they are supposed to be ignored in presence of a given portable extension.
+ * <p>
+ * CDI implementations are not required to accept custom implementations of any {@code jakarta.enterprise.lang.model}
+ * or {@code jakarta.enterprise.inject.build.compatible.spi} interface. In other words, users may only use instances
+ * of these interfaces that they previously obtained from the corresponding API. If not, non-portable behavior results.
```

Comment:
Couldn't think of a better place. We should put it to the spec text for sure, but I thought the API javadoc should have it as well.

Reply:
I assume you mean annotation values with no default values?